src/applications/mhv/landing-page/hooks/useDatadogRum.jsx

```diff
@@ -0,0 +1,35 @@
+import { useEffect } from 'react';
+import { datadogRum } from '@datadog/browser-rum';
+
+import environment from '@department-of-veterans-affairs/platform-utilities/environment';
+
+const initializeDatadogRum = () => {
+  if (
+    // Prevent RUM from running on local/CI environments.
+    environment.BASE_URL.indexOf('localhost') < 0 &&
+    // Prevent re-initializing the SDK.
+    !window.DD_RUM?.getInitConfiguration()
+  ) {
+    datadogRum.init({
+      applicationId: '1f81f762-c3fc-48c1-89d5-09d9236e340d',
+      clientToken: 'pub3e48a5b97661792510e69581b3b272d1',
+      site: 'ddog-gov.com',
+      service: 'mhv-on-va.gov',
+      env: environment.vspEnvironment(),
+      sampleRate: 100,
+      sessionReplaySampleRate: 0,
+      trackInteractions: true,
+      trackResources: true,
+      trackLongTasks: true,
+      defaultPrivacyLevel: 'mask-user-input',
```

Comment:
I noticed that the patient-check-in app was using `mask` rather than `mask-user-input`. I wonder if the former would prevent names from appearing in the va toolbar? Maybe something to keep an eye on

Reply:
Patrick and I chatted about using this mask and then using [field masking](https://docs.datadoghq.com/real_user_monitoring/session_replay/privacy_options/?tab=maskuserinput#override-an-html-element) for the name.